pom.xml

```diff
@@ -184,11 +184,11 @@
 			<scope>provided</scope>
 			<type>gem</type>
 			<exclusions>
+				<!-- we don't use this 
```

Comment:
we need to include rake now as it's not available/not preinstalled in 9.2.0.0, see jruby/jruby#5196